addons/realisticnames/stringtable.xml

```diff
@@ -1891,6 +1895,10 @@
             <Chinesesimp>MXC卡宾步枪 (黑色)</Chinesesimp>
             <Turkish>MXC (Siyah)</Turkish>
         </Key>
+        <Key ID="STR_ACE_RealisticNames_arifle_MXC_Khaki_Name">
+            <English>MXC (Khaki)</English>
+            <German>MXC (Khaki)</German>
```

Comment:
```suggestion
            <German>MXC (Khaki)</German>
            <Chinese>MXC（卡其）</Chinese>
            <French>MXC (kaki)</French>
            <Spanish>MXC (caqui)</Spanish>
            <Italian>MXC (kaki)</Italian>
            <Polish>MXC (khaki)</Polish>
            <Russian>MXC (хаки)</Russian>
            <Czech>MXC (khaki)</Czech>
            <Portuguese>MXC (cáqui)</Portuguese>
            <Korean>MXC (카키)</Korean>
            <Chinesesimp>MXC（卡其色）</Chinesesimp>
            <Japanese>MXC（カーキ）</Japanese>
            <Turkish>MXC (Haki)</Turkish>
```

Reply:
Capitalization?

Reply:
not sure, this is just based on vanilla tables
```
        <Key ID="str_a3_cfgweapons_arifle_mx_khk_f0">
          <Original>MX 6.5 mm (Khaki)</Original>
          <Chinese>MX 6.5 mm（卡其）</Chinese>
          <French>MX 6,5 mm (kaki)</French>
          <Spanish>MX 6,5 mm (caqui)</Spanish>
          <Italian>MX 6,5 mm (kaki)</Italian>
          <Polish>MX 6,5 mm (khaki)</Polish>
          <Russian>MX 6,5 мм (хаки)</Russian>
          <German>MX 6,5 mm (Khaki)</German>
          <Czech>MX 6,5 mm (khaki)</Czech>
          <English>MX 6.5 mm (Khaki)</English>
          <Portuguese>MX 6,5 mm (cáqui)</Portuguese>
          <Korean>MX 6.5 mm (카키)</Korean>
          <Chinesesimp>MX 6.5 毫米（卡其色）</Chinesesimp>
          <Japanese>MX 6.5 mm（カーキ）</Japanese>
          <Turkish>MX 6.5 mm (Haki)</Turkish>
```